src/twisted/internet/endpoints.py

```diff
@@ -1466,6 +1473,13 @@ def _parseSSL(
         dhParameters=dhParameters,
         **kw,
     )
+    if cipher:
+        try:
+            cipherBytes = cipher.replace(",", ":").encode("ascii")
```

Comment:
can this line throw an SSLError ?

Reply:
[#e9308d](https://github.com/twisted/twisted/pull/12137/commits/6e30f9f83d118633e7be1c489d00073549c1b285)